.travis.yml

```diff
@@ -18,6 +18,7 @@ node_js:
 - '4'
 before_script:
 - npm prune
+- npm update
```

Comment:
Another option would be to change all `^`s to `~`s in `package.json`.  Thoughts?